src/features/People/peopleSlice.js

```diff
@@ -13,6 +13,7 @@ const peopleSlice = createSlice({
     person: [],
     changeTileStyle: false,
     id: "undefinded",
+    isPeopleTabActive: "undefined",
```

Comment:
I think if it is "unidentified" then you don't need to write it in initial state. "Unidentified" will be the default 😉

Reply:
changed it to false because we're always statring ont the movie page